src/constants.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
module.exports is a CommonJS module system. On frontend we should ES Modules Syntax. So we should write this file either like

```js
const BOARD_SIZE_MAX = 10;
const BOARD_SIZE_MIN = 4;

exports { BOARD_SIZE_MAX, BOARD_SIZE_MIN };
```

or like this

```js
export const BOARD_SIZE_MAX = 10;
export const BOARD_SIZE_MIN = 4;
```